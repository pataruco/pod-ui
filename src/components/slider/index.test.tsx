import { shallow } from 'enzyme';
import React from 'react';
import { Carrousel, Slide } from './index';

describe('Carrousel', () => {
  it('render', () => {
    const defaultProps = {
      date: 'somedate',
      files: [
        {
          url: 'someurl',
        },
      ],
    };

    expect(shallow(<Carrousel {...defaultProps} />)).toMatchSnapshot();
  });
});

describe('Slide', () => {
  it('render', () => {
    const defaultProps = {
      date: 'somedate',
      file: {
        url: 'someurl',
      },
    };

    expect(shallow(<Slide {...defaultProps} />)).toMatchSnapshot();
  });
});
