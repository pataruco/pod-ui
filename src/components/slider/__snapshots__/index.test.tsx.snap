// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Carrousel render 1`] = `
<i
  className={null}
  currentSlide={0}
  disableAnimation={false}
  disableKeyboard={false}
  dragEnabled={true}
  hasMasterSpinner={false}
  interval={5000}
  isPageScrollLocked={false}
  isPlaying={false}
  lockOnWindowScroll={false}
  naturalSlideHeight={0}
  naturalSlideWidth={100}
  orientation="horizontal"
  playDirection="forward"
  step={1}
  tag="div"
  totalSlides={1}
  touchEnabled={true}
  visibleSlides={1}
>
  <n
    classNameTray="slider"
  >
    <n
      index={0}
      innerClassName="innerSlide"
      key="0"
    >
      <SlideWrapper
        date="somedate"
        file={
          Object {
            "url": "someurl",
          }
        }
        key="0"
      />
    </n>
  </n>
  <aside
    className="footer"
  >
    <n>
      Back
    </n>
    <n>
      Next
    </n>
  </aside>
</i>
`;

exports[`SlideWrapper render 1`] = `
<img
  alt="Peter on somedate"
  src="https://peter-of-the-day.s3.amazonaws.com/someurl"
/>
`;
